playwright.config.ts

```diff
@@ -0,0 +1,48 @@
+// eslint-disable-next-line import/no-nodejs-modules
+import path from 'node:path'
+import type {PlaywrightTestConfig} from '@playwright/test'
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: path.join(__dirname, 'test', 'playwright'),
+  testMatch: '**/*.test.ts',
+  /* Maximum time one test can run for. */
+  timeout: 30 * 1000,
+
+  // https://playwright.dev/docs/api/class-testconfig#test-config-output-dir
+  outputDir: path.join(__dirname, '.playwright', 'results'),
+  snapshotDir: path.join(__dirname, '.playwright', 'screenshots'),
+
+  /* Run tests in files in parallel */
+  fullyParallel: true,
```

Comment:
Such a good option 🔥 Did not realize that it's parallel on test files by default